src/java/org/apache/cassandra/db/DeletionTime.java

```diff
@@ -195,42 +195,95 @@ public static Serializer getSerializer(Version version)
             return legacySerializer;
     }
 
-    // Serializer for Usigned Integer ldt
+    /* Serializer for Usigned Integer ldt
+     *
+     * ldt is encoded as a uint in seconds since unix epoch, it can go up o 2106-02-07T06:28:13+00:00 only. 
+     * mfda is a positive timestamp. We use the sign bit to enconde LIVE DeletionTimes
+     */
     public static class Serializer implements ISerializer<DeletionTime>
     {
+        // We use the sign bit to signal LIVE DeletionTimes
+        private final static int IS_LIVE_DELETION = 0b1000_0000;
```

Comment:
Nit: I would prefer 0xFF as the leading byte for `LIVE`. This leaves more space if (for whatever reason) we need unsigned values or encode something else with the top bit/bits.

Reply:
Atm this is a byte we're not using at all but for one bit. Any future change to that would imply a new sstable format change where we'd free to redo things from scratch. But if this is sounding better to you I am happy to change it :-)

Reply:
I ended up reverting to `0b1000_0000`. With 0xFF the bitwise `&` operator was obviously catching all values of mfda with anything above the 56th bit. That means adding another `&` operation or affecting readability. Given you marked it as a nit and that we'd have to do a new sstable format if we wanted to change that I thought it would be ok.

Reply:
just a question - if `0b1000_0000` is used as a flag indicator in the future, shouldn't `LIVE` flag be encoded as `0b1000_0001`? If LIVE's presence is encoded as all zeros, it can be weird to encode the lack of it

Reply:
The `1` in `0b1000_0000` is what marks the presence of LIVE. The rest of the zeros are not used at all and have no meaning. Makes sense? I am very bad at explaining things sometimes. Also in the future if we change this it will come with an sstable format change for compatibility reasons afaik.

Reply:
I was under the impression that the `1` in `0b1000_0000` should be a flags byte presence marker - so it denotes that the remaining 7 bits should be interpreted as flags. Then, LIVE is one of those flags. Perhaps it does not makes much sense because it is hardly to imagine what could be stored there 🤷🏻‍♂️ Anyway, I've already approved so you do not have to bother